docs/overview/getting-started/intro.md

```diff
@@ -2,69 +2,55 @@
 slug: /intro
 ---
 
-# Overview
+# Introduction
 
-## Introduction
+Avalanche is an open-source platform for building decentralized applications (Dapps) in one
```

Comment:
```suggestion
Avalanche is an open-source platform for building decentralized applications (dApps) in one
```

Reply:
Refer to previous comment about the linter; repo enforces `Dapps` so we'd need to make a separate PR for this